docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+services:
+  omega:
+    build: .
+    depends_on:
+    - redis
+    environment:
+    - HUBOT_SLACK_TOKEN=${SLACK_TOKEN}
+  redis:
```

Comment:
@fonglh, is redis used?

Reply:
the hubot requires redis for the waffles... it seems...